apps/meteor/ee/client/voip/modal/DialPad/hooks/useEnterKey.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+
+export const useEnterKey = (onEnter: () => void, disabled: boolean): void => {
+	useEffect(() => {
+		const sendOnEnter = (e: KeyboardEvent): void => {
+			if (e.key !== 'Enter' || disabled) {
+				return;
+			}
+
+			e.stopPropagation();
+			onEnter?.();
```

Comment:
`onEnter` is required for this method, isn't it? This whole hook loses meaning if it doesn't receive `onEnter`.